### ISSUE TITLE
Threads: Only the first Slack reply gets bridged as a Matrix reply

### ISSUE THREAD

**Issue by jaller94** [matrix-org/matrix-appservice-slack#483]
### Steps to reproduce
1. Write a message in Matrix.
2. Reply in Slack, starting a thread.
3. Post again in Slack in the thread.

### Expected
The third message gets bridged to Matrix and is a reply to the second.

### Actual
The third message get bridged to Matrix but is not a reply to the second.
![Screenshot_2020-09-08_17-32-00](https://user-images.githubusercontent.com/10872136/92497116-3c5b7980-f1f9-11ea-86a8-496ed0067b27.png)

### Errors in the terminal
2nd message
```
Sep-8 17:23:07.302 INFO SlackGhost Updating user information for U010AAR88B1
Sep-8 17:23:07.314 INFO bridge [-] GET http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/event/%24SnF8sKcflRSbEwKx-6PvEQBwd8S-GnT4Rs9dN_HJ5uY (@slack_matrix-bridge-dev_U0118PJQ4CF:synapse-dev.local) Body: 
Sep-8 17:23:07.323 INFO bridge [-] GET http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/event/%24SnF8sKcflRSbEwKx-6PvEQBwd8S-GnT4Rs9dN_HJ5uY (@slack_matrix-bridge-dev_U0118PJQ4CF:synapse-dev.local) HTTP 200 "{\"type\": \"m.room.message\", \"room_id\": \"!ZmWyDEBEKfOdXjldOC:synapse-dev.l
sendEvent of type m.room.message in !ZmWyDEBEKfOdXjldOC:synapse-dev.local with txnId m1599578587324.3
Sep-8 17:23:07.324 INFO bridge [-] PUT http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/send/m.room.message/m1599578587324.3 (@slack_matrix-bridge-dev_U010AAR88B1:synapse-dev.local) Body: "{\"m.relates_to\":{\"m.in_reply_to\":{\"event_id\":\"$SnF8sKcflRSbEwKx-6PvEQBwd
Sep-8 17:23:07.359 INFO bridge [-] PUT http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/send/m.room.message/m1599578587324.3 (@slack_matrix-bridge-dev_U010AAR88B1:synapse-dev.local) HTTP 200 "{\"event_id\": \"$n7IT1e0E-c6egXLtmPraFmlZBgqBK_4TRwDmdepZRFM\"}"
Event sent to !ZmWyDEBEKfOdXjldOC:synapse-dev.local with event id $n7IT1e0E-c6egXLtmPraFmlZBgqBK_4TRwDmdepZRFM
Sep-8 17:23:07.360 ERROR BridgedRoom Failed to process event
Sep-8 17:23:07.360 ERROR BridgedRoom TypeError: Cannot read property 'event_id' of undefined
    at SlackGhost.sendMessage (/home/jaller94/Git/matrix-appservice-slack/lib/SlackGhost.js:270:62)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async SlackGhost.sendWithReply (/home/jaller94/Git/matrix-appservice-slack/lib/SlackGhost.js:300:16)
    at async BridgedRoom.handleSlackMessage (/home/jaller94/Git/matrix-appservice-slack/lib/BridgedRoom.js:632:24)
    at async BridgedRoom.onSlackMessage (/home/jaller94/Git/matrix-appservice-slack/lib/BridgedRoom.js:484:13)
    at async SlackRTMHandler.handle (/home/jaller94/Git/matrix-appservice-slack/lib/SlackEventHandler.js:54:25)
    at async RTMClient.<anonymous> (/home/jaller94/Git/matrix-appservice-slack/lib/SlackRTMHandler.js:140:21)
Sep-8 17:23:07.401 INFO bridge ::1 - - [08/Sep/2020:15:23:07 +0000] "PUT /transactions/247?access_token=<REDACTED> HTTP/1.1" 200 2 "-" "Synapse/1.15.1"
```

3rd message
```
Sep-8 17:23:20.863 INFO SlackGhost Updating user information for U010AAR88B1
sendEvent of type m.room.message in !ZmWyDEBEKfOdXjldOC:synapse-dev.local with txnId m1599578600879.4
Sep-8 17:23:20.879 INFO bridge [-] PUT http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/send/m.room.message/m1599578600879.4 (@slack_matrix-bridge-dev_U010AAR88B1:synapse-dev.local) Body: "{\"body\":\"2nd: Slack\",\"format\":\"org.matrix.custom.html\",\"formatted_body
Sep-8 17:23:20.910 INFO bridge [-] PUT http://localhost:8008/_matrix/client/r0/rooms/!ZmWyDEBEKfOdXjldOC%3Asynapse-dev.local/send/m.room.message/m1599578600879.4 (@slack_matrix-bridge-dev_U010AAR88B1:synapse-dev.local) HTTP 200 "{\"event_id\": \"$JGAHKcxRJPj1SrpPRjapkT6-IiWliXXf_G-NDZqn7PQ\"}"
Event sent to !ZmWyDEBEKfOdXjldOC:synapse-dev.local with event id $JGAHKcxRJPj1SrpPRjapkT6-IiWliXXf_G-NDZqn7PQ
Sep-8 17:23:20.911 ERROR BridgedRoom Failed to process event
Sep-8 17:23:20.911 ERROR BridgedRoom TypeError: Cannot read property 'event_id' of undefined
    at SlackGhost.sendMessage (/home/jaller94/Git/matrix-appservice-slack/lib/SlackGhost.js:270:62)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async BridgedRoom.onSlackMessage (/home/jaller94/Git/matrix-appservice-slack/lib/BridgedRoom.js:484:13)
    at async SlackRTMHandler.handle (/home/jaller94/Git/matrix-appservice-slack/lib/SlackEventHandler.js:54:25)
    at async RTMClient.<anonymous> (/home/jaller94/Git/matrix-appservice-slack/lib/SlackRTMHandler.js:140:21)
Sep-8 17:23:20.943 INFO bridge ::1 - - [08/Sep/2020:15:23:20 +0000] "PUT /transactions/248?access_token=<REDACTED> HTTP/1.1" 200 2 "-" "Synapse/1.15.1"

```

**Comment by Half-Shot** [comment]
This was a bug caused by depending on a buggy version of matrix-appservice-bridge which did not pass the event_id down, depending on >=1.13.2 fixes the bug.

**Comment by jaller94** [comment]
Looks like `docker pull matrixdotorg/matrix-appservice-slack:ems-build-10` still had this bug.
Redeploying their bridge with the `latest` docker image of the slack bridge solved the issue for them.
Go have some fun on your vacation.

**Comment by jaller94** [comment]
Fixed by 53172b9814e4efeb69be316003a7744f7ded930b.
